fastmri/data/mri_data.py

```diff
@@ -125,6 +127,7 @@ def __init__(
         challenge,
         sample_rate=1,
         dataset_cache_file=pathlib.Path("dataset_cache.pkl"),
+        num_cols=None
```

Comment:
This needs to be added to `CombinedSliceDataset` as well.

Reply:
Added, can you check it makes sense?